community-pulse/app/schemas/question.py

```diff
@@ -11,6 +10,10 @@ class Config:
         from_attributes = True
 
 
+class CategoryBase(BaseModel):
+    name: str
+
+
 class CategoryCreate(BaseModel):
     name: str
 
```

Comment:
Если у  вас есть базовая абстрактная модель категории, которая содержит поле `name`, то все остальные модели, которые содержат тоже поле могут просто наследоваться от класса базовой абстрактной модели. Точно также как у вас сделано для вопросов.
```suggestion
class CategoryBase(BaseModel):
    name: str


class CategoryCreate(CategoryBase):
    pass

```
Позже можно кастомизировать как абстрактную родительскую модель, так и дочерние